File: 백준/Bronze/25305. 커트라인/커트라인.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.Collections;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int k = Integer.parseInt(br.readLine().split(" ")[1]);
        Integer[] scores = Arrays.stream(br.readLine().split(" ")).map(Integer::valueOf).toArray(Integer[]::new);

        Arrays.sort(scores, Collections.reverseOrder());
        System.out.println(scores[k - 1]);
    }
}

File: 백준/Bronze/25305. 커트라인/커트라인.js
const fs = require('fs');
const inputs = fs.readFileSync('./dev/stdin').toString().trim().split('\n');

const awardCriteria = inputs[0].split(" ");
const scores = inputs[1].split(" ");

scores.sort((a, b) => b - a);
console.log(scores[awardCriteria[1] - 1]);